data/Templates/Stu3ToR4/metadata.json

```diff
@@ -1,3 +1,3 @@
 {
-	"type": "json"
+	"type": "fhir"
```

Comment:
Shall we distinguish `fhir-stu3` from `fhir-r4`? Though currently we only need r3->r4 transformation, but possibly we need r4->r5 in the future?

Reply:
Reasons not include version information in datatype:
1. If add version here, there will be too many types in the future. And most of these datatypes will share same logic and same engine.
2. Include version here such as "fhir-stu3", "fhir-r4" are not at the same level with previous data types ("hl7v2"and "ccda").
3. For the current metadata format, it's not sufficient to present entire version information. Based on our previous assumption, converter's output is FHIR R4 format. If r4->r5 added, we needs to re-design the format of metadata to represent both input data type and output data type. 

For these three reasons, we add "fhir" here to distinguish FHIR resources with general JSON, and this data type is the same level with "hl7v2" and "ccda". 

If we really need to distinguish different versions in the future, there are two ways:
1. Re-design metadata structure: add new sub-field to represent input and output version if data type is "fhir". As for converter's API, add new parameters for input and output version.
2. Add version information into templates, and make it transparent for converter engine.

Reply:
Got it. Thanks